app/web_framework/cluster_base_handler.py

```diff
@@ -38,8 +38,10 @@ def _handle_request_exception(self, ex):
         :param ex: The exception that was caught
         :type ex: Exception
         """
-        # _handle_request_exception() is called in the exception handler, so we can still use logger.exception.
-        self._logger.exception('Exception occurred during request to {}.', self.request.uri)
+        # Requests to access an item that isn't ready shouldn't log anything
+        if not isinstance(ex, ItemNotReadyError):
```

Comment:
Good idea, that could be here as well. The other types of exceptions seem like they'd be important to have in the logs, so do we just want to `ItemNotReadyError` and `ItemNotFoundError`?